perf/benchmark/configs/nomixer_latency.yaml

```diff
@@ -0,0 +1,21 @@
+# Configuration Set4: Latency Quantiles with mixer disabled
+mixer_mode: "nomixer"
+conn:
+    - 1
+    - 2
+    - 4
+    - 8
+    - 16
+    - 32
+    - 64
+qps:
+    - 1000
+duration: 240
+size: 1024
+metrics:
+    - p50
+    - p90
+    - p99
+perf_record: true
```

Comment:
let's mark perf_record all to be "false" until flame graph being successfully generated?

Reply:
should be able to kept as true if the run today succeed

Reply:
the ssh is success, but another issue pops up:
```Unable to use a TTY - input is not a terminal or the right kind of file
OCI runtime exec failed: exec failed: container_linux.go:345: starting container process caused "exec: \"/etc/istio/proxy/get_perfdata.sh\": stat /etc/istio/proxy/get_perfdata.sh: no such file or directory": unknown
command terminated with exit code 126```.  Let me look into it.